### PR TITLE
MMCA-4895 | In-service notification on Cash account dashboard

### PR DESCRIPTION
**Pull request by Neozxc**
Task:
- Query SDES data for Cash Account CSV files in the cash account frontend.
- If the requested statements are returned in the SDES response, display the notification panel.
- Add unit tests / amend existing ones.

### CODE REVIEW

**Comment by AdamCoops** [comment]
[info] *** 10 TESTS FAILED ***
[error] Failed tests:
[error] 	controllers.CashAccountV2ControllerSpec

File: test/connectors/SdesConnectorSpec.scala [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+      when(requestBuilder.setHeader(any[(String, String)]())).thenReturn(requestBuilder)
+
+      when(requestBuilder.execute(any[HttpReads[Seq[FileInformation]]], any[ExecutionContext]))
+        .thenReturn(Future.successful(cashStatementFilesWithUnknownFileTypesSdesResponse))
+
+      when(mockHttp.get(eqTo(url"$sdesCashStatementUrl"))(any()))
+        .thenReturn(requestBuilder)
+
+      running(app) {
+        val result = await(sdesConnector.getCashStatements(someEori)(hc, messages))
```
Comment (tyagikumarvineet) [inline test/connectors/SdesConnectorSpec.scala:53]: Better to make use of map instead of await

File: test/models/FileFormatSpec.scala [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+class FileFormatSpec extends SpecBase {
+
+  "FileFormat" should {
+
+    "compare FileFormats correctly" in {
+      Pdf < Csv mustBe true
+      Csv < UnknownFileFormat mustBe true
+      UnknownFileFormat > Pdf mustBe true
+      Pdf compare Csv must be <0
```
Comment (tyagikumarvineet) [inline test/models/FileFormatSpec.scala:31]: Could you please format from line 31-33 ?

File: test/viewmodels/CashAccountV2ViewModelSpec.scala
Comment (tyagikumarvineet) [inline test/viewmodels/CashAccountV2ViewModelSpec.scala:1]: Could you please add an assertion to check the cashStatementNotification value?

Better to add as part of method like it has been done for other fields

File: test/views/CashAccountV2Spec.scala
Comment (tyagikumarvineet) [inline test/views/CashAccountV2Spec.scala:1]: Could you please add assertion for the check of existence of notification panel?

**Comment by tyagikumarvineet** [comment]
As discussed, all these comments would be addressed as part of separate ticket.

**Review by tyagikumarvineet — APPROVED** [review]
Some of the tests would be taken care as part of separate ticket. Approving

**Review by tyagikumarvineet — APPROVED** [review]

**Pull request merged**